mods/d2k/weapons.yaml

```diff
@@ -455,24 +455,21 @@ DeviatorMissile:
 	ReloadDelay: 160
 	Range: 5c0
 	Report: MISSLE1.WAV
-	InvalidTargets: Infantry, Structure
 	Projectile: Missile
 		MaximumLaunchSpeed: 281
-		RangeLimit: 40
```

Comment:
Removing this means it might fly infinitely (until it leaves the map) when it loses its target. I agree 40 might be too short, but something in the 50-75 range (=2-3 seconds at normal gamespeed) should be enough to prevent it from exploding too early.